aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/ReadWriteFileStepTest.java

```diff
@@ -49,4 +50,33 @@
         r.assertLogContains("HELLO", r.assertBuildStatusSuccess(p.scheduleBuild2(0)));
     }
 
+	@Test
+	public void shouldTestFileExistsAndNotFindFile() throws Exception
+	{
+        final WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition(
+                "node {\n" +
+                "  def exists = fileExists('test.txt')  \n" +
+                "  echo \"FileExists: ${exists}\" \n" +
+                "}"));
+
+		r.assertLogContains("FileExists: false", p.scheduleBuild2(0).get());
+		r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+    }
+
+	@Test
+	public void shouldTestFileExistsAndFindFile() throws Exception
+	{
+		final WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+		p.setDefinition(new CpsFlowDefinition(
+				"node {\n" +
+				"  writeFile file: 'test2.txt', text:'content of file' \n" +
+				"  def exists = fileExists('test2.txt')  \n" +
+				"  echo \"FileExists: ${exists}\" \n" +
```

Comment:
Could save a few seconds of test time and shorten sources by merging these two test cases into one: `fileExists`, `echo`, `writeFile`, `fileExists`, `echo` should print false then true.